benchmark/scenarios/_request_load.py

```diff
@@ -128,10 +128,19 @@ def handle_request_error(result):
             write_failure(result)
 
         for i in range(self.request_rate):
+            t0 = self.reactor.seconds()
+
             d = self.scenario_setup.make_request()
+
+            def get_time(_ignore):
+                return self.reactor.seconds() - t0
+            d.addCallback(get_time)
+
             self.rate_measurer.request_sent()
-            d.addCallbacks(self.rate_measurer.response_received,
-                           errback=handle_request_error)
+            d.addCallbacks(
+                self.rate_measurer.response_received,
+                handle_request_error
+            )
```

Comment:
I realize this isn't introduced in this branch, but, what's up with the Deferred not being returned?